docs/source/advanced/boosted_frame.rst

```diff
@@ -274,23 +262,20 @@ the plasma wake phase velocity, i.e. :math:`\gamma_{wake} \approx 42`.
 .. image:: ../images/boosted_frame_speedup_example.svg
 
 This simple example highlights two important aspects of choosing the right
-:math:`\gamma_{boost}` in practice. First, it can be seen that the speedup does
-only follow the simple scaling law :math:`(1+\beta_{boost}^2)\gamma_{boost}^2`
-for :math:`\gamma_{boost} \ll \gamma_{wake}`, and second, that the optimum speedup
-occurs at a :math:`\gamma_{boost} < \gamma_{wake}` before the simulation
+:math:`\gamma_b` in practice. First, it can be seen that the speedup does
+only follow the simple scaling law :math:`(1+\beta_b^2)\gamma_b^2`
+for :math:`\gamma_b \ll \gamma_{wake}`, and second, that the optimum speedup
+occurs at a :math:`\gamma_b < \gamma_{wake}` before the simulation
 slows down again.
 
 .. note::
 
     As highlighted in the above example, there is an optimum
-    :math:`\gamma_{boost}` for which the interaction time is minimized for a
+    :math:`\gamma_b` for which the interaction time is minimized for a
     given simulation setup. The speed up of a boosted frame
-    simulation will reverse for too high values of :math:`\gamma_{boost}`.
-    In practice, it is often advisable to choose :math:`\gamma_{boost}`
-    such that :math:`L_{plasma}' \approx l_{window}'` in the boosted frame,
-    i.e. the plasma entirely fits into the simulation box. Additionally, the
-    condition :math:`\gamma_{boost} \ll \gamma_{wake}` should be satisfied -
-    as explained in the next section.
+    simulation will reverse for too high values of :math:`\gamma_b`.
+    In practice, it is often advisable to choose :math:`\gamma_b < \gamma_{wake}/2`
+    - as explained in the next section.
```

Comment:
I am not sure if `L_{plasma}' \approx l_{window}'` is a general and robust enough relation that it can be put into the documentation. Any opinion?

Reply:
offline discussion: 

* gamma_boost 5 to 10 good choice in practice
* plasma should be longer than window